src/components/organisms/metadata/index.tsx

```diff
@@ -61,7 +61,7 @@ const Metadata: React.FC<AddMetadataProps> = ({
       <div className="flex flex-col mt-base gap-y-base">
         {localData.map((field, index) => {
           return (
-            <DeletableElement key={index} onDelete={deleteKeyPair(index)}>
+            <DeletableElement key={field.key} onDelete={deleteKeyPair(index)}>
```

Comment:
Not an expert on this, but I think we need to use a constant value for the key prop. The `field.key` value might change when editing the metadata so maybe this is not that appropriate?

Reply:
@srindom using the index here had a bug where if you deleted an item, then it would show the wrong values. This seemed to fix that issue.